swssconfig/sample/00-copp.config.json

```diff
@@ -30,6 +30,18 @@
         "OP": "SET"
     },
     {
+        "COPP_TABLE:trap.group.neigh_discovery": {
```

Comment:
is it v6 neighbor discovery? move this to the arp group as there are the same nature?